mllib/src/test/scala/org/apache/spark/mllib/classification/LogisticRegressionSuite.scala

```diff
@@ -462,6 +464,42 @@ class LogisticRegressionSuite extends FunSuite with MLlibTestSparkContext with M
 
   }
 
+  test("model save/load") {
+    // NOTE: This will need to be generalized once there are multiple model format versions.
+    val nPoints = 20
+    val A = 2.0
+    val B = -1.5
+
+    val testData = LogisticRegressionSuite.generateLogisticInput(A, B, nPoints, 42)
+    val testRDD = sc.parallelize(testData, 2)
+    testRDD.cache()
+
+    val lr = new LogisticRegressionWithLBFGS().setIntercept(true)
+    lr.optimizer.setNumIterations(1)
+    val model = lr.run(testRDD)
```

Comment:
Shall we construct a model directly? A unit test for model save/load should not trigger training.